templates/components/base.jinja2

```diff
@@ -19,9 +18,10 @@
             {% endblock %}
         </main>
         {% endblock %}
-
+        <script src="/static/theme.js"></script>
+        <script src="https://kit.fontawesome.com/822c3cc402.js" crossorigin="anonymous"></script>
```

Comment:
what is this? can we integrate this locally without adding one more js dependency?